xudo/commands/build.py

```diff
@@ -21,27 +19,27 @@ def run(self):
 
         if self.options["-c"]:
             os.chdir(javaPath)
-            os.system("docker rm api_changelog_1")
```

Comment:
the `api_` part just comes from whatever directory the docker-compose file is in.

So now its `hbng_`

Reply:
oh didn't know this thanks.